teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(specialization, age, name, parent_permission)
+    super(age, name, parent_permission)
+    @specialization = specialization
+  end
```

Comment:
- [ ] Same here, default values for name and parent_permission should be assigned here same way you did it in Person class, otherwise you will be getting argument error.